Snakefile

```diff
@@ -16,9 +16,54 @@ def read_metadata_summary():
 
 read_metadata_summary()
 
-#to be implemented
+
+def get_methods_template_baseline():
+    if 'methods_base' in config:
+        return config['methods_base']
+    else:
+        return None
+
+def get_methods_template_differential():
+    if 'methods_dif' in config:
+        return config['methods_dif']
+    else:
+        return None
+
+def get_zooma_exclusions():
+    if 'zooma_exclusions' in config:
+        return config['zooma_exclusions']
+    else:
+        return None
+
 def get_isl_dir():
-    return None
+    if 'isl_dir' in config:
+        return config['isl_dir']
+    else:
+        return None
+
+def get_isl_genomes():
+    if 'isl_genomes' in config:
+        return config['isl_genomes']
+    else:
+        return None
+
+def get_irap_versions():
+    if 'irap_versions' in config:
+        return config['irap_versions']
+    else:
+        return None
+
+def get_irap_container():
+    if 'irap_container' in config:
+        return config['irap_container']
+    else:
+        return None
+
+def get_tmp_dir():
+    if 'tmp_dir' in config:
+        return config['tmp_dir']
+    else:
+        return None
```

Comment:
Unfortunately, currently mktemp won't create a temp dir name in appropriate codon path. But I have included this option in case we do not want a custom TMP_DIR 18bd548